tests/roots/test-ext-autodoc/target/cython.pyx

```diff
@@ -0,0 +1,12 @@
+# cython: binding=True
+
+def foo(*args, **kwargs):
+    """Docstring."""
```

Comment:
Coincidentally, this is the only one passed correctly for functions.  All pure Python functions defined in Cython has the same signature, e.g. https://mcsinyx.github.io/palace/html/reference.html#palace.sample_size

The definition of those are here: https://github.com/McSinyx/palace/blob/master/src/palace.pyx#L175

Reply:
Thank you for the info. I found my implementation does not work well.